deploy/tests/000037-legacy-staking-rewards-claim.ts

```diff
@@ -0,0 +1,108 @@
+import { AccessControlEnumerable, BancorNetworkInfo, PoolToken } from '../../components/Contracts';
+import { BNT, StakingRewardsClaim } from '../../components/LegacyContracts';
+import { expectRoleMembers, Roles } from '../../test/helpers/AccessControl';
+import { describeDeployment } from '../../test/helpers/Deploy';
+import { DeployedContracts, isMainnet } from '../../utils/Deploy';
+import { merkleRoot } from '../scripts/000037-legacy-staking-rewards-claim';
+import { expect } from 'chai';
+import { BigNumber } from 'ethers';
+import { ethers } from 'hardhat';
+import MerkleTree from 'merkletreejs';
+
+const {
+    utils: { getAddress, solidityKeccak256, keccak256 }
+} = ethers;
+
+interface Reward {
+    claimable: string;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const snapshot: Record<string, Reward> = require('@bancor/contracts-solidity/snapshot/snapshot.json');
+
+describeDeployment(__filename, () => {
+    let bnt: BNT;
+    let bnBNT: PoolToken;
+    let networkInfo: BancorNetworkInfo;
+    let stakingRewardsClaim: StakingRewardsClaim;
+    let merkleTree: MerkleTree;
+
+    before(async () => {
+        merkleTree = new MerkleTree(
```

Comment:
You may as well do that at declaration (e.g., `const merkleTree = new MerkleTree...`).

Semantic: can also declare `generateLeaf` next to it (instead of after the `beforeEach`).

Reply:
Sure